lib/hx509/cert.c

```diff
@@ -3403,6 +3403,9 @@ hx509_cert_binary(hx509_context context, hx509_cert c, heim_octet_string *os)
 #define __attribute__(X)
 
 void
+#ifdef _WIN32
+_declspec(noreturn)
```

Comment:
Missing underscore?

Reply:
And whitespace issue?

Reply:
Also, why are we `#define`ing a no-op `__attribute__()` macro above, at 3403??  Shouldn't we be doing that only in the `_WIN32` case??